packages/batch-submitter/src/exec/run-batch-submitter.ts

```diff
@@ -16,19 +16,34 @@ import {
   TX_BATCH_SUBMITTER_LOG_TAG,
 } from '..'
 
+const environment = process.env.NODE_ENV
+const network = process.env.ETH_NETWORK
+const release = `batch-submitter@${process.env.npm_package_version}`
+
 /* Logger */
 const name = 'oe:batch_submitter:init'
-const log = new Logger({
-  name,
-  sentryOptions: {
-    release: `batch-submitter@${process.env.npm_package_version}`,
-    dsn: process.env.SENTRY_DSN,
-    tracesSampleRate: 0.05,
-  },
-})
+let logger
+
+if (network) {
+  // Initialize Sentry for Batch Submitter deployed to a network
+  logger = new Logger({
+    name,
+    sentryOptions: {
+      release,
+      dsn: process.env.SENTRY_DSN,
+      tracesSampleRate: parseInt(process.env.SENTRY_TRACE_RATE, 10) || 0.05,
```

Comment:
Would we ever want to set this to 0?

Reply:
@tynes I equate that to not deploying sentry right now

Reply:
It would disable tracing, but we primary need sentry for error tracking, so setting it to 0 would still keep error tracking enabled.